"""
Exceptions module
"""

class DatabaseError(Exception):
    """Errors concerning communication with local database"""
